apps/condo/domains/common/utils/smsAdapter.js

```diff
@@ -0,0 +1,216 @@
+const { isEmpty, get, has } = require('lodash')
```

Comment:
please move it inside notifications utils

Reply:
Ok

Reply:
Done